epsilons/main_online_experiments.py

```diff
@@ -19,7 +19,7 @@
 from agents.implicit_quantile_agent_new import *
```

Comment:
why is this file in a folder called "epsilons"? isn't this more general than just epsilon experiments?